nova/objects/migrate_data.py

```diff
@@ -503,3 +503,58 @@ def from_legacy_dict(self, legacy):
         for field in self.fields:
             if field in legacy:
                 setattr(self, field, legacy[field])
+
+
+@obj_base.NovaObjectRegistry.register
+class VMWareLiveMigrateData(LiveMigrateData):
+    # Version 1.0: Initial version
+    # Version 1.1: Added
```

Comment:
Is this comment missing the important part?

Reply:
True, explained now what I added, and for what use-case.